pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -69,7 +69,9 @@ type BindMechanism interface {
 	startDHCP(vmi *v1.VirtualMachineInstance) error
 }
 
-type podNICImpl struct{}
+type podNICImpl struct {
+	driver NetworkHandler
```

Comment:
but why is it still a `driver` rather a `handler`, @qinqon. I don't know if it is a github quirk that hides my question from you :-(.

Reply:
This is old, is no longer handler, please review latest version.

Reply:
oh, I think it's because a wrong split to commits. You introduce `driver` in one, and change it to `handler` in another - which confused me.

Beside this, I think this PR is ok (but I'm not sure I'm qualified to really approve this grand change)

Reply:
Done